### PR TITLE
[CentOS] update installation instructions for NodeJS

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Updates the installation command for NodeJS and so version 8 will be installed.

#### Testing instructions (if applicable)

1. Test command on CentOS

#### Links to related tickets (GitHub, Redmine, ...)

* https://github.com/aces/Loris/issues/5725#issuecomment-558259292

### CODE REVIEW

File: README.CentOS7.md [hunk 1]
```diff
@@ -78,7 +78,8 @@ Then follow instructions to create a password for the root user.
 
 ## NodeJS
 ```bash
-sudo yum install nodejs
+curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
+sudo yum install -y nodejs
```
Comment (PapillonMcGill) [inline README.CentOS7.md]:
```suggestion

```
This line seam to be not needed.  Script downloaded and ran at line 81, install the RPM.

Comment (maltheism) [reply README.CentOS7.md]: @PapillonMcGill oops, thank you!

File: README.CentOS7.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ Then follow instructions to create a password for the root user.
 
 ## NodeJS
 ```bash
-sudo yum install nodejs
+curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
```
Comment (coreyAbout) [inline README.CentOS7.md]: Running this line does not finish the installation of nodejs, such as output:
"## Run `sudo yum install -y nodejs` to install Node.js 8.x LTS Carbon and npm."
To fix, after this line, please add the following:
`sudo yum install -y nodejs`

Comment (maltheism) [reply README.CentOS7.md]: @coreyAbout updated.

Comment (PapillonMcGill) [reply README.CentOS7.md]: No, this is just reversing the previous change and will install nodejs version that come with CentOS.
If you don't believe me, look at the script from previous line and also https://github.com/nodesource/distributions/blob/master/README.md

Comment (maltheism) [reply README.CentOS7.md]: @PapillonMcGill oops, fixed.

**Comment by coreyAbout** [comment]
@PapillonMcGill Do you mind to remind me why we need to remove the line 
`sudo yum install -y nodejs` ? 
After running `curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -`, this line will work fine.

**Comment by coreyAbout** [comment]
Check out my log @PapillonMcGill 
```
[yc@localhost ~]$ curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -

## Installing the NodeSource Node.js 8.x LTS Carbon repo...


## Inspecting system...

+ rpm -q --whatprovides redhat-release || rpm -q --whatprovides centos-release || rpm -q --whatprovides cloudlinux-release || rpm -q --whatprovides sl-release
+ uname -m

## Confirming "el7-x86_64" is supported...

+ curl -sLf -o /dev/null 'https://rpm.nodesource.com/pub_8.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm'

## Downloading release setup RPM...

+ mktemp
+ curl -sL -o '/tmp/tmp.nK8KEogcUH' 'https://rpm.nodesource.com/pub_8.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm'

## Installing release setup RPM...

+ rpm -i --nosignature --force '/tmp/tmp.nK8KEogcUH'

## Cleaning up...

+ rm -f '/tmp/tmp.nK8KEogcUH'

## Checking for existing installations...

+ rpm -qa 'node|npm' | grep -v nodesource

## Run `sudo yum install -y nodejs` to install Node.js 8.x LTS Carbon and npm.
## You may also need development tools to build native addons:
     sudo yum install gcc-c++ make
## To install the Yarn package manager, run:
     curl -sL https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yum.repos.d/yarn.repo
     sudo yum install yarn

[yc@localhost ~]$ sudo yum install -y nodejs
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
epel/x86_64/metalink                                                                                                                                                                  |  17 kB  00:00:00     
 * base: mirror.its.dal.ca
 * epel: mirror.math.princeton.edu
 * extras: mirror.calgah.com
 * remi-safe: mirror.cedia.org.ec
 * updates: centos.mirror.colo-serv.net
base                                                                                                                                                                                  | 3.6 kB  00:00:00     
epel                                                                                                                                                                                  | 5.4 kB  00:00:00     
extras                                                                                                                                                                                | 2.9 kB  00:00:00     
mariadb                                                                                                                                                                               | 2.9 kB  00:00:00     
nodesource                                                                                                                                                                            | 2.5 kB  00:00:00     
remi-safe                                                                                                                                                                             | 3.0 kB  00:00:00     
updates                                                                                                                                                                               | 2.9 kB  00:00:00     
(1/3): nodesource/x86_64/primary_db                                                                                                                                                   |  48 kB  00:00:00     
(2/3): epel/x86_64/primary_db                                                                                                                                                         | 6.9 MB  00:00:00     
(3/3): epel/x86_64/updateinfo                                                        | 1.0 MB  00:00:06     
Resolving Dependencies
--> Running transaction check
---> Package nodejs.x86_64 2:8.16.2-1nodesource will be installed
--> Finished Dependency Resolution

Dependencies Resolved

============================================================================================================
 Package             Arch                Version                              Repository               Size
============================================================================================================
Installing:
 nodejs              x86_64              2:8.16.2-1nodesource                 nodesource               16 M

Transaction Summary
============================================================================================================
Install  1 Package

Total download size: 16 M
Installed size: 49 M
Downloading packages:
warning: /var/cache/yum/x86_64/7/nodesource/packages/nodejs-8.16.2-1nodesource.x86_64.rpm: Header V4 RSA/SHA512 Signature, key ID 34fa74dd: NOKEY
Public key for nodejs-8.16.2-1nodesource.x86_64.rpm is not installed
nodejs-8.16.2-1nodesource.x86_64.rpm                                                 |  16 MB  00:00:00     
Retrieving key from file:///etc/pki/rpm-gpg/NODESOURCE-GPG-SIGNING-KEY-EL
Importing GPG key 0x34FA74DD:
 Userid     : "NodeSource <gpg-rpm@nodesource.com>"
 Fingerprint: 2e55 207a 95d9 944b 0cc9 3261 5ddb e8d4 34fa 74dd
 Package    : nodesource-release-el7-1.noarch (installed)
 From       : /etc/pki/rpm-gpg/NODESOURCE-GPG-SIGNING-KEY-EL
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : 2:nodejs-8.16.2-1nodesource.x86_64                                                       1/1 
  Verifying  : 2:nodejs-8.16.2-1nodesource.x86_64                                                       1/1 

Installed:
  nodejs.x86_64 2:8.16.2-1nodesource                                                                        

Complete!
[yc@localhost ~]$ node --version
v8.16.2
[yc@localhost ~]$ npm --version
6.4.1

**Comment by PapillonMcGill** [comment]
Effectively, the script from nodesource.com only install a new repo for NodeJS, not NodeJS itself. Followed the instruction at the top of the script and README.md. `yum install nodejs` is required.

**Comment by maltheism** [comment]
@PapillonMcGill @coreyAbout is everything alright now with the latest commit?

**Comment by coreyAbout** [comment]
@PapillonMcGill This confuses me. What you said seems that you agree with what I said (we need the line `sudo yum install -y nodejs` to install nodejs).
However, I suggested to add the line `sudo yum install -y nodejs` after the line `curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -`, but you disagreed and removed it. That's the part I do not get. 
cc @maltheism

**Comment by PapillonMcGill** [comment]
I made a mistake, not reading the entire README.md page not analysing the entire script.
Yes, we need `sudo yum install -y nodejs` after the `curl ...`line.

**Comment by maltheism** [comment]
@PapillonMcGill @coreyAbout thank you both for the help. I added the line back.

**Comment by coreyAbout** [comment]
OK cool @PapillonMcGill

**Pull request merged**